kloppy/infra/serializers/tracking/skillcorner.py

```diff
@@ -209,7 +209,7 @@ def _get_skillcorner_attacking_directions(cls, frames, periods):
 
         return attacking_directions
 
-    def __load_json(self, file):
+    def __load_json_raw(self, file):
```

Comment:
The refactored function looks a bit messy to me.

1. Should we avoid code duplication by moving the timestamp fix into an auxiliary function?
2. Do we need the `__load_json` auxiliary function? In the `if` branch, the `json.loads` function is called directly.

Reply:
I agree that it was a bit messy, latest commit should've addressed all your concerns.

- removed `__load_json`, not sure why it was there to begin with
- added `__replace_timestamp`